project/ws/app/src/lib/routes/events/routes/list/list-event.component.ts

```diff
@@ -132,7 +132,8 @@ export class ListEventComponent implements OnInit, AfterViewInit, OnDestroy {
                         eventDuration: duration,
                         eventjoined: (creatorDetails !== undefined && creatorDetails.length > 0) ?
                             ((creatorDetails.length === 1) ? '1 person' : `${creatorDetails.length} people`) : ' --- ',
-                        eventThumbnail: obj.appIcon && (obj.appIcon !== null || obj.appIcon !== undefined) ? obj.appIcon :
+                        eventThumbnail: obj.appIcon && (obj.appIcon !== null || obj.appIcon !== undefined) ?
```

Comment:
you should use this appIcon and convert it to public URL, check the karmayogi portal for that method where It'll construct that path with new variables